.github/workflows/test.yml

```diff
@@ -0,0 +1,34 @@
+name: Test LocalGovDrupal Guides
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Clone drupal_container
+        uses: actions/checkout@v2
+        with:
+          repository: localgovdrupal/drupal-container
+          ref: master
+
+      - name: Create LocalGovDrupal project
+        run: composer create-project --stability dev localgovdrupal/localgov-project html
+
+      - name: Checkout localgov_guides branch to test
+        uses: actions/checkout@v2
+        with:
+          path: html/web/modules/contrib/localgov_guides
```

Comment:
Then would it be possible to composer require the appropriate branch being tested here? It's a variable in the action I think.